api/src/main/java/org/apache/commons/rdf/api/RDFTerm.java

```diff
@@ -37,4 +55,46 @@
      */
     String ntriplesString();
 
+	/**
+	 * Check it this RDFTerm is equal to another RDFTerm.
+	 * <p>
+	 * If this object is an {@link IRI}, equality is checked using
```

Comment:
If this must _always_ be true, sounds like an opportunity to use a `default` method here.